src/Google.Cloud.Diagnostics.Debug/Google.Cloud.Diagnostics.Debug/LoggingClient.cs

```diff
@@ -130,5 +130,28 @@ private string SubstituteLogMessageFormat(string messageFormat, List<Debugger.V2
 
             return result;
         }
+
+        /// <summary>
+        /// Formats the variable into a more readable string,
+        /// especially if the variable only has members and no value.
+        /// </summary>
+        /// <returns>Formatted string representing the variable.</returns>
+        private string FormatVariable(Debugger.V2.Variable variable)
```

Comment:
Should we add more info than this? 

Maybe "Name (type): Value|Members"?  And if there is an error we should add that?